build/Targets/Imports.targets

```diff
@@ -118,6 +118,12 @@
     </When>
   </Choose>
 
+  <!-- Settings for localization -->
+  <PropertyGroup>
+    <!-- Only turn on localization for shipping binaries. -->
+    <EnableXlfLocalization Condition="'$(NonShipping)' == 'true'">false</EnableXlfLocalization>
```

Comment:
What is the intended use case? Should we be building sattellite assemblies on every single build including developer machines? Interested to know the change in build time for Roslyn.sln if we do that. If it's significant we should consider disabling on developer machines as we do a few other expensive tasks that don't impact majority of our dev lives.

Reply:
The idea behind the current implementation is to both update the XLF files and generate the satellite assemblies on every local build. CI builds won't regenerate the XLF files but will error if they are out of date. The idea is to make sure the satellite assemblies are always up-to-date with the set of resources in the original .resx files.

I've made updates to Microsoft.NET.Sdk such that the `CoreGenerateSatelliteAssemblies` task will use the shared compiler server; this greatly reduces the time required. I haven't done a before-and-after comparison, but locally a clean build with those changes takes 2.5 minutes.